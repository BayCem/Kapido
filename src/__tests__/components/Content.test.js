import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Content from '../../components/ContentComponent/Content';

configure({ adapter: new Adapter() });
it('renders coorectly', () => {
  const wrapper = shallow(
    <Content>
      <h1>Content</h1>
    </Content>,
  );
  expect(wrapper).toMatchSnapshot();
});
