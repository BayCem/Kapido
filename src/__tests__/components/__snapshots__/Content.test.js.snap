// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders coorectly 1`] = `
<div>
  <h1>
    Content
  </h1>
</div>
`;
